src/File/FrostealthS3ObjectStorageClient.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace CottaCush\Yii2\File;
+
+use Aws\S3\Exception\S3Exception;
+use frostealth\yii2\aws\s3\Storage;
+use Yii;
+use yii\helpers\ArrayHelper;
+use yii\log\Logger;
+
+/**
+ * Class S3ObjectStorageClient
+ * @author Adeyemi Olaoye <yemi@cottacush.com>
+ * @package CottaCush\Yii2\File
+ */
+class FrostealthS3ObjectStorageClient implements ObjectStorageClientInterface
+{
+    public function put($resource, $fileName)
+    {
+        try {
+            /** @var Storage $s3Bucket */
+            $s3Bucket = Yii::$app->get('s3bucket');
```

Comment:
@goke-epapa Any ideas on how to parameterize it?

Reply:
Not at the moment, let's leave as is for now, we can always revisit. Thanks